operate/view-configure-runtime-parameters.mdx

```diff
@@ -19,8 +19,8 @@ For example, you may see a table similar to this:
 ```bsh Runtime Parameters
           Name                  |     Setting     |        Description
 --------------------------------+-----------------+--------------------------------------
- rw_implicit_flush              | false           | If `RW_IMPLICIT_FLUSH` is on, then every INSERT/UPDATE/DELETE statement will block until the entire dataflow is refreshed.
- create_compaction_group_for_mv | false           | If `CREATE_COMPACTION_GROUP_FOR_MV` is on, dedicated compaction groups will be created in MV creation.
+ implicit_flush                 | false           | If set to `true`, then every INSERT/UPDATE/DELETE statement will block until the entire dataflow is refreshed.
+ create_compaction_group_for_mv | false           | If set to `true`, dedicated compaction groups will be created in MV creation.
```

Comment:
Please reword it to use active voice: something like "...RisingWave will create dedicated compaction for materialized views when creating these materialized views."